src/components/framework/fine-print.js

```diff
@@ -97,8 +90,6 @@ class FinePrint extends React.Component {
       <FinePrintStyles>
         <div style={{width: width}}>
           <Flex className='finePrint'>
-            {this.getUpdated()}
-            {dot}
             {this.downloadDataButton()}
             {dot}
```

Comment:
Why remove it from the footer? I'd think to leave it there so folks who are used to finding it there still can. There's no harm in having it in both places.

Reply:
That's fair. Updated in [force-push](https://github.com/nextstrain/auspice/compare/21ddfc802f41dc281eac9a46d0bb2b51d0b7fc34..8bcbe1a9dd26bd211a22b49a786fae73311239ea).

If there was a lot of clutter in that are of the footer I'd think to remove it, but right now it's pretty sparse.